pgjdbc/src/main/java/org/postgresql/jdbc/EscapedFunctions2.java

```diff
@@ -60,8 +60,9 @@ public static Method getFunction(String functionName) {
     if (method != null) {
       return method;
     }
+    //FIXME: this probably should not use the US locale
```

Comment:
It should use us locale

Reply:
What happens if the server is using a different locale ?

Reply:
We are looking for case insensitive matching of java method names. I would expect the behavior of that matching to be consistent regardless of locale of the OS where the code is executing.

Reply:
fair enough